Dockerfile

```
@@ -0,0 +1,23 @@
+FROM node:18-bullseye
```

Let's use the Alpine image for security and smaller resource usage:

```suggestion
FROM node:18-alpine
```